packages/quantic/package.json

```diff
@@ -11,28 +11,32 @@
     "url": "git+https://github.com/coveo/ui-kit"
   },
   "scripts": {
-    "test": "npm run test:lint && npm run test:typings && npm run test:unit",
-    "test:lint": "eslint force-app/main/default/lwc/ --format junit -o reports/eslint.xml",
+    "test": "npm run build:staticresources && npm run lint:check:tests && npm run validate:types && npm run test:unit",
+    "lint:check:tests": "eslint force-app/main/default/lwc/ --format junit -o reports/eslint.xml",
     "lint:fix": "eslint force-app/main/default/lwc/ --fix && eslint force-app/examples/main/lwc/ --fix",
     "fix": "npm run lint:fix",
-    "copy:staticresources": "node copy-static-resources.js",
-    "build": "npm run copy:staticresources",
-    "start": "npx rimraf .localdevserver && npm run copy:staticresources && sfdx force:source:deploy -p force-app/main && sfdx force:lightning:lwc:start --port 3334",
+    "build:staticresources": "node copy-static-resources.js",
+    "build": "npm run build:staticresources && npm run build:doc",
+    "dev": "npx rimraf .localdevserver && npm run build:staticresources && npm run dev:sfdx",
+    "dev:sfdx": "sfdx force:source:deploy -p force-app/main && sfdx force:lightning:lwc:start --port 3334",
     "test:unit": "../../node_modules/.bin/lwc-jest",
     "test:unit:debug": "lwc-jest --debug",
     "test:unit:watch": "lwc-jest --watch",
     "test:unit:coverage": "lwc-jest --coverage",
-    "test:typings": "tsc",
-    "cypress:open": "npx cypress open",
-    "cypress:test": "npx cypress run --browser chrome",
-    "cypress:test:detailed": "npx cypress run --browser chrome --headless --reporter cypress/reporters/detailed-reporter.js",
+    "validate:types": "tsc",
+    "e2e:watch": "npx cypress open",
+    "e2e": "npx cypress run --browser chrome",
+    "e2e:detailed": "npx cypress run --browser chrome --headless --reporter cypress/reporters/detailed-reporter.js",
```

Comment:
maybe we should always run in detailed mode?

Reply:
For some reason, I thought the CI was using `e2e:detailed` but I was wrong.

Good point about re-running it locally for details, I'll keep `e2e:detailed` and `e2e` separated then.